mne_bids/read.py

```diff
@@ -863,14 +871,15 @@ def get_head_mri_trans(bids_path, extra_params=None, t1_bids_path=None,
     mri_coords_dict = t1w_json.get('AnatomicalLandmarkCoordinates', dict())
 
     # landmarks array: rows: [LPA, NAS, RPA]; columns: [x, y, z]
+    suffix = f"_{kind}" if kind else ""
     mri_landmarks = np.full((3, 3), np.nan)
     for landmark_name, coords in mri_coords_dict.items():
-        if landmark_name.upper() == 'LPA':
+        if landmark_name.upper() == ('LPA' + suffix).upper():
```

Comment:
Shouldn't we be case-sensitive when it comes to the suffix?

```suggestion
        if landmark_name.upper() == 'LPA' + suffix
```

Reply:
Fair enough, I missed that

Reply:
should be fixed now